draft-ietf-tls-tls13.md

```diff
@@ -1375,12 +1368,9 @@ For example:
        } V2;
 
        struct {
-           select (VariantTag) {
-               case apple:
-                 V1;   /* VariantBody, tag = apple */
-               case orange:
-               case banana:
-                 V2;   /* VariantBody, tag = orange or banana */
+           select (VariantTag) { /* value of selector is implicit */
```

Comment:
Not sure that I understand this comment.  I'd remove it.  I realize that the expression "VariantTag" isn't understood, but it isn't critical to understanding.  Or, you could change this to be more explicit:

```
struct {
    VariantTag type;
    select (VariantRecord.type) {
```